python/paddle/v2/framework/tests/test_conv2d_op.py

```diff
@@ -0,0 +1,100 @@
+import unittest
+import numpy as np
+from gradient_checker import GradientChecker, create_op
+from op_test_util import OpTestMeta
+from paddle.v2.framework.op import Operator
+
+
+class TestConv2dOp(unittest.TestCase):
```

Comment:
The operator python test framework has been refactored, please merge develop branch and change the unit test accordingly.

Reply:
Done.